includes/admin/meta-boxes/views/html-order-fee.php

```diff
@@ -1,32 +1,30 @@
 <?php
 /**
- * Shows an order item fee
+ * Shows an order item fee.
  *
- * @var object $item The item being displayed
- * @var int $item_id The id of the item being displayed
+ * @var object $item The item being displayed.
  */
-
 if ( ! defined( 'ABSPATH' ) ) {
 	exit;
 }
 ?>
-<tr class="fee <?php echo ( ! empty( $class ) ) ? $class : ''; ?>" data-order_item_id="<?php echo $item_id; ?>">
+<tr class="fee <?php echo ( ! empty( $class ) ) ? $class : ''; ?>" data-order_item_id="<?php echo $item->get_id(); ?>">
```

Comment:
Both class and ID should be escaped on output (per best practices, even if they are safe).